.github/workflows /Ofrp.yml

```
@@ -0,0 +1,167 @@
+        max-size: "10G"
+      
+    - name: Set-up Manifest # For github workspace use single {}
+      run: |
+        mkdir -p ${GITHUB_WORKSPACE}/OrangeFox
+        cd ${GITHUB_WORKSPACE}/OrangeFox
+        git config --global user.name "ツ๛abrohim๛"
+        git config --global user.email "callofduty.pubg786@gmail.com
+        git clone https://gitlab.com/OrangeFox/sync.git -b master
+        cd sync
+        ./orangefox_sync.sh --branch ${{ inputs.MANIFEST_BRANCH }} --path ${GITHUB_WORKSPACE}/OrangeFox/fox_${{ inputs.MANIFEST_BRANCH }}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)